.github/release/standard/Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV TERM linux
 
 RUN apt-get update
-RUN apt-get --yes install git
```

Comment:
When looking at this again, I wonder why even `git` would be required here. Maybe it was needed in a previous version to get hold of the repository. If you agree, please remove it altogether.